sc/social/like/utils.py

```diff
@@ -198,3 +200,30 @@ def validate_og_lead_image(image):
         raise ValueError(MSG_INVALID_OG_LEAD_IMAGE_ASPECT_RATIO)
 
     return True
+
+
+def validate_og_fallback_image(value):
```

Comment:
please add a `TODO` comment to move code to share this logic with `validate_og_lead_image` method

Reply:
I was thinking about it, but it's not so easy as the code only looks similar.